modules/services/model/build.gradle.kts

```diff
@@ -2,14 +2,13 @@ plugins {
     alias(libs.plugins.android.library)
     alias(libs.plugins.kotlin.android)
     alias(libs.plugins.ksp)
+    alias(libs.plugins.room)
 }
 
 android {
     namespace = "au.com.shiftyjelly.pocketcasts.model"
-    defaultConfig {
-        ksp {
-            arg("room.schemaLocation", "$projectDir/schemas")
-        }
+    room {
+        schemaDirectory("$projectDir/schemas")
```

Comment:
This fixes ksp task input by normalizing it (not relying on absolute path for arguments). Not the easiest change to find. This how arguments differ between CI and local:

<img width="2272" alt="Screenshot 2025-04-02 at 16 08 21" src="https://github.com/user-attachments/assets/44907960-e9df-4a55-8e66-14d7e50276ee" />

After applying the Gradle Plugin, schema directory is provided by some other mechanism that uses absolute path (I didn't deep dive into internals).